targets/arduino_101/scripts/get-zephyr.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Copyright 2016 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#sudo apt-get update -q
+#sudo apt-get install -q -y \
+# dependencies
+
+# Clone zephyr to deps
+# git clone https://gerrit.zephyrproject.org/r/zephyr 
```

Comment:
This file contains only comment lines. Please remove it you don't need it or uncomment the necessary parts.